.github/workflows/conventional-commit.yml

```diff
@@ -0,0 +1,15 @@
+name: PR Title Checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+
+jobs:
+  validate-pr-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Conventional Commit Validation
+        uses:  ytanikin/pr-conventional-commits@1.4.0
+        with:
+          task_types: '["feat","fix", "docs", "test", "ci", "refactor", "chore", "revert"]'
```

Comment:
❓ : Would general updates like to wording count as a `chore` or `fix`?

Reply:
I think it depends on what general updates do - if it is a fix for something I would prefix it with`fix:`.  This cheatsheet might be helpful: https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13#types